### DISCUSSION TITLE
Design System - Storybook Adding Addons

### DISCUSSION THREAD

**Discussion by jrock2004** [vercel/turborepo#4432] [Help] [answered]
So I used the template, change the acme to another name. Now I want to add some storybook book plugins, but when I install them and try to run dev, it won't build storybook? Is there something special I need to do or missing?

![image](https://user-images.githubusercontent.com/655716/229383644-e27fa2c3-52ab-411f-8b72-156333c05921.png)

**Comment by anthonyshew** [comment, marked as answer]
I think this question would be better equipped to ask in Storybook resources.

If you think there is a problem with Turborepo specifically, please open an issue. 🙏
